storage/allocator.go

```diff
@@ -35,21 +37,16 @@ const (
 	// target and it will always be eligible to rebalance replicas to other
 	// stores.
 	maxFractionUsedThreshold = 0.95
-	// minFractionUsedThreshold: if the mean fraction used of a list of store
-	// descriptors is less than this, then range count will be used to make
-	// rebalancing decisions instead of the fraction of bytes used. This is
-	// useful for distributing load evenly on nascent deployments.
-	minFractionUsedThreshold = 0.02
-	// rebalanceFromMean is used to declare a range above and below the average
-	// used capacity of the cluster. If a store's usage is below this range, it
-	// is a rebalancing target and can accept new replicas; if usage is above
-	// this range, the store is eligible to rebalance replicas to other stores.
-	rebalanceFromMean = 0.025 // 2.5%
 
 	// priorities for various repair operations.
 	removeDeadReplicaPriority  float64 = 10000
 	addMissingReplicaPriority  float64 = 1000
 	removeExtraReplicaPriority float64 = 100
+
+	// minSecsBetweenRebalances is the minimum number of seconds that must elapse
+	// between replica rebalances. This should be slightly larger than
+	// server.gossipStoresInterval.
+	minRebalanceInterval = 65 * time.Second
```

Comment:
I'm not sure I believe this will do the right thing. Imagine a 3-node cluster where each node has 100 replicas. Then a 4th node is added. We want to move 25 replicas from each existing node (75 replicas total) "quickly". With this setting, "quickly" will take ~30 min if the range leadership is evenly spread amongst the existing nodes and ~80 min if it is unbalanced. Another perspective: 75 replicas at 64 MB per replicas is ~700 MB of data. We should be able to rebalance that much data in a handful of minutes.